### ISSUE TITLE
This action is broken - command arguments are ignored resulting in errors

### ISSUE THREAD

**Issue by antoine-sac** [sanity-io/github-action-sanity#16]
This github action does not work at all (at least on ubuntu-latest) because there is an error in the entrypoint. All commands taking arguments are impacted.

When you use args like `. dataset export production backup.tar.gz`, the command getting executed is only `sanity dataset` which of course doesn't do anything but complain that the usage is not correct.

The reason is this line in `entrypoint.sh`:

```bash
sh -c "cd $1 && sanity install && SANITY_AUTH_TOKEN=$SANITY_AUTH_TOKEN sanity ${@:2:99}"
```

The problem is that `${@:2:99}` is not compatible with basic shells like /bin/sh. To confirm it, you can create a simpler entrypoint showing what it executes:

```bash
sh -c "echo SANITY_AUTH_TOKEN=$SANITY_AUTH_TOKEN sanity ${@:2:99}"
```

which gives, on my machine:

```
$ ./entrypoint.sh . dataset export production backup.tar.gz
SANITY_AUTH_TOKEN=REDACTED sanity dataset
```

In summary : everything after the first command is ignored which breaks all workflows.

A simple fix is to use the `shift` operator which is standard shell:

```bash
#!/bin/bash

set -e

sh -c "cd $1 && sanity install"
shift
sh -c "SANITY_AUTH_TOKEN=$SANITY_AUTH_TOKEN sanity $*"
```

Another fix is to simply not call `sh -c` but run the commands directly.

**Comment by antoine-sac** [comment]
This is the root cause of #8 and #12
